web_client/src/django.ts

```diff
@@ -52,7 +52,11 @@ const djangoClient = {
     try {
       await apiClient.post('/logout/', undefined, { withCredentials: true });
     } finally {
-      await oauthClient.logout();
+      await oauthClient.logout().then(
+        async () => {
+          await oauthClient.redirectToLogin();
```

Comment:
Are we sure this doesn't cause an infinite loop? I remember when refactoring this before that that would happen under certain conditions.

Reply:
What we had there before causing the infinite loop was `window.location.reload`. In my testing, this did not loop.

Reply:
Ok, we can deploy this and I'll do some testing on the demo site to make sure no loop occurs.